src/components/templates/TalkShowStudio/TalkShowStudio.tsx

```diff
@@ -47,9 +32,19 @@ export const TalkShowStudio: FC<TalkShowStudioProps> = ({ venue }) => {
   const stage = useStage();
   const { userId, profile } = useUser();
   const currentVenue = useSelector(currentVenueSelectorData);
-  const remoteUsers = useAgoraRemotes({ client: remotesClient });
   const isRequestToJoinStageEnabled = venue.requestToJoinStage;
 
+  const [remotesClient, screenClient, cameraClient] = React.useMemo(() => {
+    return Array.from({ length: 3 }).map(() =>
```

Comment:
This `Array.from({ length: 3 })` is neat by the way 🎉 . I was going to call it out in my review as probably being invalid, but then when I went and looked it up and thought about it for a while, it made sense:

> array-like objects (**objects with a length property** and indexed elements); or

I also learned that `Array.from` has a built in `mapFn` param that I also never knew existed.

> `Array.from()` has an optional parameter `mapFn`, which allows you to execute a `map()` function on each element of the array being created.
> 
> More clearly, `Array.from(obj, mapFn, thisArg)`
> has the same result as `Array.from(obj).map(mapFn, thisArg)`,
> except that it does not create an intermediate array.

(cc // @sunny-viktoryia FYI: Another contender for our 'quickest way to make an array' benchmarking challenge :p )

Reply:
wooow! Super cool! haven't heard about `mapFn` either.
hahahha I've just found our discussion about that benchmarking https://github.com/sparkletown/sparkle/pull/1307/files#r608258420